### DISCUSSION TITLE
Support for Multi-user and OIDC

### DISCUSSION THREAD

**Discussion by ankycooper** [louislam/dockge#475] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

enable support for Multi-user and OIDC via ext IDP

### ✔️ Solution

enable support for Multi-user and OIDC via ext IDP

**Comment by NekoSuneVR** [comment]
Multi user be good, because that way like Developers will have they own account by selected role permissons. example


Main admin will have super admin perm with full rights to site
have roles like

Admin as mini same power as superadmin but not full control stuff
Developers as they can access deploy and stuff
Viewer as they can view console but cant touch nothing then see as read only thing only.

but Multi accounts should be higher priority because most of us dont trust have single account. and never know single account be easy hacked and abuse down line. so make have 2fa too for this.

**Reply by NekoSuneVR** [reply]
sorry no response for months / years after feeling this wasnt going anywhere after 3 years so i decide fork they code and build this on myself. including features that was in pull request in here this github was never accepted and been pending for years

so i decide intergrate SEO stuff and other tweeks on it. but still early access my fork and not sure official owner will comply all this intergrations help us control more in docker

https://github.com/NekoSuneProjects/dockge

see post here snippets but added more too in it recently too https://github.com/louislam/dockge/discussions/936

i woudlnt recomended use my version if dont want risk it since little buggy that i am slowly fixing it when i have other projects ongoing atm while reason why forked dockge and added my own features my fork because trying move away from portainer pay to use stuff. when dockge could be easy replaceable portainer features as i did mine fork.

but not recomended less really want switch and not saying do it or not. but feeling i should keep maintain peoples request till official owner decide add all this features for us. when my fork already way head of everything was requested so far.

**Comment by Rumpick** [comment]
+1
OIDC support and permission controls are different features. 
I would like OIDC support so I don't have to keep track of myriad credentials, but permissions aren't necessary for small setups, which is probably more common for Dockge.

**Comment by rohit-purandare** [comment]
+1 to OIDC support, would love to have PocketID connected

**Reply by satr14washere** [reply]
+1

**Comment by SnorreSelmer** [comment]
Would love to have SSO via Authentik OAuth2, yes!

**Comment by PaRkThEcAr** [comment]
+1 for OIDC support! UptimeKuma has it, so this would be pretty sweet to have it here too. especially the dangerous rights it has.
